graylog2-server/src/main/java/org/graylog/integrations/ipfix/IpfixParser.java

```diff
@@ -54,7 +54,7 @@
  * </p>
  */
 public class IpfixParser {
-    private static final RateLimitedLog LOG = getRateLimitedLog(IpfixParser.class);
+    private static final RateLimitedLog LOG = createDefaultRateLimitedLog(IpfixParser.class);
```

Comment:
Sorry for the trouble with this one. The PR fixing this on `master` is now merged, so this should be fixed when merging in master.